### PR TITLE
Show error on giving up trying to connect

### PR DESCRIPTION
**Pull request by HelloUniverseNo31415**
# Checklist:

Related work items: #275 

Preview url: [N/A]

- [ ] Tests have been added for new functionality
- [ ] Work has been reviewed by other team members
- [x] Work has been reviewed by @craigayre or @eclennett

### CODE REVIEW

File: src/components/StoreList/actions.js [hunk 1]
```diff
@@ -18,10 +18,11 @@ export function setLoadStoreListError(loadStoreListError) {
   };
 }
 
-export const loadStoreList = () => dispatch =>
-  retry(fetchItems, () => dispatch(setLoadStoreListError(true)))
+export const loadStoreList = history => dispatch => {
+  return retry(fetchItems, () => dispatch(setLoadStoreListError(true)))
```
Comment (eclennett) [inline src/components/StoreList/actions.js]: why did you add the return?
don't need it if you remove the `{` brakets

File: src/components/App/App.js [hunk 2]
```diff
@@ -63,16 +64,14 @@ class App extends Component {
     this.firebaseAuth.onAuthStateChanged(user =>
       this.setState({loggedIn: user !== null})
     );
-
+
```
Comment (eclennett) [inline src/components/App/App.js]: added random whitspace

File: src/components/App/App.js [hunk 3]
```diff
@@ -101,39 +100,17 @@ class App extends Component {
...
-        this.props.sendMessageError) &&
-      this.state.loggedIn
-    );
-  }
-
   componentWillUnmount() {
     clearTimeout(this.timer);
     clearInterval(this.interval);
     document.body.removeEventListener('touchstart', this.resetTimeoutTimer);
     document.body.removeEventListener('touchmove', this.resetTimeoutTimer);
```
Comment (cainy393) [inline src/components/App/App.js]: My fault as it was me that added event listeners for mouse move and scroll but they need removing on unmount too.
